src/core.js

```diff
@@ -489,10 +489,16 @@ $.extend( $.validator, {
 			this.submitted = {};
 			this.prepareForm();
 			this.hideErrors();
-			var i, elements = this.elements()
+			var elements = this.elements()
 				.removeData( "previousValue" )
 				.removeAttr( "aria-invalid" );
 
+			this.resetElements( elements );
+		},
+
+		resetElements: function( elements ) {
```

Comment:
I guess not, as there are a lot of methods added to prototype (which means published) but not documented anywhere in the source code nor the documentation.

But if there is a better place for this method, please let me know.